Earthfile

```diff
@@ -61,4 +61,6 @@ edit-docs:
 # check-lint-openapi - OpenAPI linting from a given directory
 check-lint-openapi:
```

Comment:
This target should be in an earthfile in `examples/openapi` not the root of cat-ci.